bookshelf.js

```diff
@@ -0,0 +1,46 @@
+var button = document.createElement("button");
+button.setAttribute("id","fetch-books-btn");
+var text = document.createTextNode("Fetch Books");
+button.appendChild(text);
+
+
+var BE = document.getElementsByTagName('body')[0];
+BE.appendChild(button);
+
+button.addEventListener("click",fetchBooks);
+
+function fetchBooks(){
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+    fetch(booksJSON)
+        .then(response => response.json())
+        .then(processBooks => {
```

Comment:
The spec says to replace this inline function with a functio called `processBooks`. Here you're naming the argument `processBooks` not the function